src/graph/test/SchemaTest.cpp

```diff
@@ -313,7 +381,42 @@ TEST_F(SchemaTest, metaCommunication) {
         };
         ASSERT_TRUE(verifyResult(resp, expected));
     }
-
+    // Test multi sentences
+    {
+        cpp2::ExecutionResponse resp;
+        std::string query;
+        for (auto i = 0u; i < 1000; i++) {
+            query += "CREATE TAG tag10" + std::to_string(i) + "(name string);";
+        }
+        auto code = client->execute(query, resp);
+        ASSERT_EQ(cpp2::ErrorCode::SUCCEEDED, code);
+    }
+    {
```

Comment:
let's suppose tag100~tag101000 have been created.
then let's
1. drop tag10100~tag10300. 
2. alter tag10400~tag10500
let's check the results.
1. -> not existed
2. -> changed
3. -> others, remain unchanged.